en/support/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Version Support
+menu: support
+lang: en
+---
+
+# Version Support
+
+Only the latest version of any given major release line is supported.
```

Comment:
```suggestion
Only Express 4 and Express 5 are currently supported.
```

The original sentence is true only because v5 is not released yet.  We plan to support v4 for some time after v5 is out, and later support v5 after v6 is out, etc.

It may be easier/simpler to remove this line entirely.

Reply:
It’s true even when 5 comes out - it’s not saying which majors are supported (that’s in the table), but when v5.1 is out, v5.0 is unsupported

Reply:
ah I see.  maybe the text can be clearer, but I'll also acquiesce if this is adjudged to be a _me_ problem